core/domain/role_services.py

```diff
@@ -65,7 +72,8 @@
 ACTION_SUBSCRIBE_TO_USERS = 'SUBSCRIBE_TO_USERS'
 ACTION_SUGGEST_CHANGES = 'SUGGEST_CHANGES'
 ACTION_UNPUBLISH_ANY_PUBLIC_ACTIVITY = 'UNPUBLISH_ANY_PUBLIC_ACTIVITY'
-ACTION_VISIT_ANY_TOPIC_EDITOR = 'VISIT_ANY_TOPIC_EDITOR'
+ACTION_VISIT_ANY_QUESTION_EDITOR = 'VISIT_QUESTION_EDITOR'
```

Comment:
Might as well have the string match the constant name (missing "ANY").

Reply:
Done